src/storage/file_io.c

```diff
@@ -6749,21 +6749,21 @@ fileio_initialize_backup (const char *db_full_name_p, const char *backup_destina
   /* currently, disable the following code. DO NOT DELETE ME NEED FUTURE OPTIMIZATION */
   fileio_determine_backup_buffer_size (session_p, buf_size);
 #endif
-  if ((int) prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE) > 0
-      && (session_p->bkup.iosize >= MIN ((int) prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE), DB_INT32_MAX)))
+  if (prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE) > 0
+      && (session_p->bkup.iosize >= MIN (prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE), DB_INT32_MAX)))
```

Comment:
Can the MIN macro be removed?

Reply:
Because need to limit for iosize below INT_MAX, I think, can't remove MIN macro.